tests/dummy/app/pods/docs/guide/migration-4-0-to-5-0/template.md

```diff
@@ -55,3 +55,20 @@ or
 
 `{{format-number numCustomers notation="compact"}}`
 
+## **New Features**
+
+### **Filter Locales**
```

Comment:
Not something that should go in the migration docs.  Can you find a new home for it in the docs - maybe a new section for config options (I likely should have added one but doesn't look like I have)?

I can fill in the other config options before versioning.

Reply:
Ok, i will update docs